DataFormats/EcalDigi/src/EcalLiteDTUSample.cc

```diff
@@ -0,0 +1,9 @@
+#include "DataFormats/EcalDigi/interface/EcalLiteDTUSample.h"
+#include <iostream>
+
+EcalLiteDTUSample::EcalLiteDTUSample(int adc, int gainId) { theSample = (adc & 0xFFF) | ((gainId & 0x3) << 12); }
```

Comment:
See comment in header file. Use named constants for bit masks and bit shifts.

Reply:
Implemented with named constants